src/python/py_wallet.rs

```diff
@@ -29,6 +29,17 @@ const MAIN_PUBKEY_HASH: u8 = 0x00;
 const TEST_PUBKEY_HASH: u8 = 0x6f;
 
 // TODO: note only tested for compressed key
+// Given a WIF, return bytes rather than SigningKey
+pub fn wif_to_bytes(wif: &str) -> Result<Vec<u8>>{
+    // Ok(private_key) => Ok(private_key),
```

Comment:
Does this need to be in here?  If not, delete the comment?

Reply:
Did you mean the dead-code? If so removed.